src/dql_resolution.erl

```diff
@@ -8,7 +8,9 @@
 %%%-------------------------------------------------------------------
 -module(dql_resolution).
 
--export([resolve/2, propagate/1, start_time/1]).
+% -export([resolve/2, propagate/1, start_time/1]).
+
+-compile([export_all]).
```

Comment:
not automatically but you can set it in the rebar.config for example makeing a test-shell profile